cms/static/js/views/active_video_upload_list.js

```diff
@@ -162,13 +184,43 @@ define(
                 if (this.onFileUploadDone) {
                     this.onFileUploadDone(this.collection);
                     this.clearSuccessful();
+                    this.fileErrorMsg = null;
                 }
             },
 
             fileUploadFail: function(event, data) {
                 this.setStatus(data.cid, ActiveVideoUpload.STATUS_FAILED);
             },
 
+            showErrorMessage: function(error) {
+                this.fileErrorMsg = new NotificationView.Error({
+                    title: gettext('Your file could not be uploaded'),
+                    message: error
+                });
+                this.fileErrorMsg.show();
+            },
+
+            validateFile: function(data) {
+                var error, fileName, fileType;
+
+                $.each(data.files, function(index, file) {  // eslint-disable-line consistent-return
+                    fileName = file.name;
+                    fileType = fileName.substr(fileName.lastIndexOf('.') + 1);
+                    // validate file type
+                    if (supportedFileFormats.indexOf(fileType) === -1) {
+                        error = gettext('`{filename}` has unsupported file format.')
+                                        .replace('{filename}', fileName);
+                        return false;
```

Comment:
Can we show message here and return from here? If yes then we can avoid the `if(error)` below.